### ISSUE TITLE
数据集尺寸问题

### ISSUE THREAD

**Issue by terhh996** [sstary/SSRS#47]
![2024-10-30 17-46-07 的屏幕截图](https://github.com/user-attachments/assets/ff6cddd6-0290-412b-956a-4f918d156f75)
数据集我换成了自己的数据集，尺寸为256×256，tif格式的图片，标签为同样尺寸格式的二分黑白图片，用于二分类
现在存在这样的问题
能麻烦你帮我看一下吗

LABELS = ["background", "land"]
    N_CLASSES = len(LABELS)  # Number of classes
    WEIGHTS = torch.ones(N_CLASSES)  # Weights for class balancing
    MAIN_FOLDER = FOLDER + 'Urban/'
    DATA_FOLDER = MAIN_FOLDER + 'image/{}.tif'
    LABEL_FOLDER = MAIN_FOLDER + 'label/{}.tif'
    ERODED_FOLDER = MAIN_FOLDER + 'label/{}.tif'
    palette = {0 : (0, 0, 0), # background
           1 : (255, 255, 255)} # land
    invert_palette = {v: k for k, v in palette.items()}
